sugarcrm/tests/test_customer.py

```diff
@@ -0,0 +1,38 @@
+from unittest import TestCase
+from unittest.mock import MagicMock, patch
+from sugarcrm.customer import Customer
+from api.tasks import _check_customer_is_partner
+
+
+class CustomerTest(TestCase):
+    def setUp(self):
+        self.mock_client = MagicMock()
+        self.email = "email@email.com"
+        self.roles_from_crm_as_partner = "^is_partner^,^is_role_test^"
+        self.roles_from_crm_not_as_partner = "^is_not_partner^,^is_role_test^"
+
+    def test_customer_is_partner(self):
+        customer = Customer()
+        customer.api_client = self.mock_client
```

Comment:
are setters implicitly available in all Python classes? :astonished:

Reply:
yeah in python is quite flexible about that. Maybe is not that beauful but for test I think is ok... anyway i've added api_client for the constructor parameter which looks better.